public/data/ckvignesh.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "C K Vignesh",
+  "bio": "Programmer",
```

Comment:
Maybe add a bit more of yourself, what do you do, your passions, etc.?

Reply:
Sure, will do it.

Reply:
I mean that isn't _necessary_, so this is good to go :)

Reply:
Thank you, @Vyvy-vi